internal/host/validator.go

```diff
@@ -737,51 +739,50 @@ func (v *validator) hasSufficientPacketLossRequirementForRole(c *validationConte
 		return ValidationSuccess
 	}
 
+	v.log.Debugf("Maximum packet loss percentage allowed for host %s: %f", c.host.ID, *c.clusterHostRequirements.Total.PacketLossPercentage)
```

Comment:
I guess that you added it for debug - please remove it. this method is called every 8 seconds

Reply:
Same comment as before: is the default log level set to debug in production systems? If not, this line won't have any impact.

Reply:
Removed.